File: qc-core/src/main/java/com/eager/questioncloud/core/domain/payment/infrastructure/repository/QuestionPaymentJpaRepository.java
package com.eager.questioncloud.core.domain.payment.infrastructure.repository;

import com.eager.questioncloud.core.domain.payment.infrastructure.entity.QuestionPaymentEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface QuestionPaymentJpaRepository extends JpaRepository<QuestionPaymentEntity, String> {
    int countByUserId(Long userId);
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/payment/infrastructure/repository/QuestionPaymentJpaRepository.kt
package com.eager.questioncloud.core.domain.payment.infrastructure.repository

import com.eager.questioncloud.core.domain.payment.infrastructure.entity.QuestionPaymentEntity
import org.springframework.data.jpa.repository.JpaRepository

interface QuestionPaymentJpaRepository : JpaRepository<QuestionPaymentEntity, String> {
    fun countByUserId(userId: Long): Int
}
